uportal-war/src/main/java/org/jasig/portal/portlets/portletadmin/PortletAdministrationHelper.java

```diff
@@ -118,13 +112,14 @@
 
 /**
  * Helper methods for the portlet administration workflow.
- * 
+ *
  * @author Jen Bourey, jbourey@unicon.net
  * @revision $Revision$
  */
 @Service
 public class PortletAdministrationHelper implements ServletContextAware {
-	protected final Log logger = LogFactory.getLog(PortletAdministrationHelper.class);
+
+    protected final Log logger = LogFactory.getLog(PortletAdministrationHelper.class);
```

Comment:
Agreed. Also, added final to the class as this is a helper class that is not intended to be subclassed.